src/routes/PublicLayoutRouter/PublicLayoutRouter.js

```diff
@@ -28,9 +28,10 @@ export default function PublicLayoutRouter (props) {
         <Route path={`/${POST_DETAIL_MATCH}`} exact component={PublicPostDetail} />
         <Route path='/:context(groups)/:groupSlug' exact component={PublicGroupDetail} />
         <Route path='/:context(public)/:view(map)' component={MapExplorerLayoutRouter} />
-        <Route path='/:context(public)/:view(groups)' exact component={GroupExplorerLayoutRouter} />
+        <Route path='/:context(public)/:view(groups)(.*)' exact component={GroupExplorerLayoutRouter} />
```

Comment:
Did you try removing `exact`? I think that would do the same thing and be more idiomatic.